automatic/flashplayerppapi/update.ps1

```diff
@@ -0,0 +1,41 @@
+
+import-module au
+
+$releases = "https://get.adobe.com/en/flashplayer/" # URL to for GetLatest
+
+function global:au_BeforeUpdate {
+  # We need this, otherwise the checksum won't get created
+  # Since windows 8 or later is skipped.
+  $Latest.ChecksumType32 = 'sha256'
+  $Latest.Checksum32     = Get-RemoteChecksum $Latest.URL32
+}
+
+function global:au_SearchReplace {
+  @{
+    ".\tools\chocolateyInstall.ps1" = @{
+      "(^[$]version\s*=\s*)('.*')"= "`$1'$($Latest.Version)'"
+      "(^[$]majorVersion\s*=\s*)('.*')"= "`$1'$($Latest.majorVersion)'"
```

Comment:
the version and majorVersion can be removed, since they are pretty much unneccessary in the install script